### DISCUSSION TITLE
建议支持图片缩放

### DISCUSSION THREAD

**Discussion by Moriefy** [WatsonYao/nian_app#123] [Q&A] [answered]
您好！我想提出一个关于图片预览功能的小小建议。
在使用Nian的过程中，我发现在查看图片时，有时需要放大以更清晰地查看细节，比如较小的文字或图案。目前，虽然Nian支持在计本中插入图片并预览，但似乎还不支持放大功能，这在某些情况下可能会稍显不便。
所以希望能在图片预览界面支持双手缩放或者双击缩放。谢谢！

**Comment by WatsonYao** [comment, marked as answer]
既然系统有了相册带了极其丰富的图片操作功能，nian这里本着精简、不重复代码的原则，就没有提供图片方面的操作功能。确实有一些不方便，但已经是取舍利弊后的结果了。
